src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -129,10 +199,19 @@ class TableView extends React.Component<TableViewProps> {
    * Please read the comment on `_createColumn`
    */
   _moveColumnCommit = (fromIndex: number, toIndex: number) => {
-    const {location, eventView} = this.props;
+    const {location, eventView, organization} = this.props;
 
     const nextEventView = eventView.withMovedColumn({fromIndex, toIndex});
 
+    // metrics
+    trackAnalyticsEvent({
+      eventKey: 'discover_v2.move_column',
+      eventName: 'Discoverv2: A column was moved',
+      from_index: fromIndex,
+      to_index: toIndex,
+      organization_id: organization.id,
+    });
+
```

Comment:
It'll be useful to track which column key is being moved. Same reason as deleting.

Reply:
I've confirmed with Bowen on this; he's fine with not having to track this.

Reply:
This event will say, "A columns was moved from index 5 to index 0" which isn't useful information. And that will lead to the question, "I wonder which column...?".

IMO if we're collecting occurrences of any event to draw product insights, it should have high information density.

Reply:
Good point. I've added `aggregation`, `field`, and `fieldname`.